app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -64,13 +64,13 @@ def toggle_status
     assign_conversation if @conversation.status == 'open' && Current.user.is_a?(User) && Current.user&.agent?
   end
 
+  def toggle_priority
+    @conversation.toggle_priority(params[:priority])
+  end
+
   def toggle_typing_status
-    case params[:typing_status]
-    when 'on'
-      trigger_typing_event(CONVERSATION_TYPING_ON, params[:is_private])
-    when 'off'
-      trigger_typing_event(CONVERSATION_TYPING_OFF, params[:is_private])
-    end
+    typing_status_manager = ::Conversations::FilterService.new(@conversation, current_user, params)
```

Comment:
Did you mean `Conversations::TypingStatusManager`

Reply:
Oh yeah, fixing it